docs/getting-started/dockstore-tools.rst

```diff
@@ -92,23 +85,16 @@ Sharing Your Tool
 .. this section is purposely not symmetric with its equivalent in tool-legacy-registration.rst
 
 After you have successfully added your tool onto Dockstore and have it
-synced with GitHub, Bitbucket, or GitLab, you are now ready to share your
+synced with GitHub, Bitbucket, or GitLab, or hosted directly in Dockstore, you are now ready to share your
 tool with the public! Assuming that your tool has at least one valid
 version, you can publish your tool for everyone to use. Simply select the
-tool on the ``/my-tool`` page and click publish.
-
-Find Other Tools
-----------------
-
-You can find tools on the Dockstore website or also through the
-``dockstore tool search`` command line option.
```

Comment:
It is worth mentioning the CLI's search feature appears to be documented nowhere else, but I made a ticket for that (https://github.com/dockstore/dockstore/issues/4835) and would be alright with removing this section.

Reply:
(Not requesting any changes here, just making note that this functionality is not mentioned elsewhere, in case that influences whether or not you decide to keep the deletion -- totally fine with either outcome.)